io.github.efogdev.mpris-timer.yml

```diff
@@ -0,0 +1,38 @@
+id: io.github.efogdev.mpris-timer
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: mpris-timer
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=~/.var/app:create
```

Comment:
why is this needed?

Reply:
The app needs access to some kind of persistent cache dir

Reply:
it already has access to its own cache dir in `~/.var/app/$app-id/cache`.

Reply:
fixed